src/engine/Match_search_mode.ml

```diff
@@ -617,6 +619,14 @@ let rec filter_ranges (env : env) (xs : (RM.t * MV.bindings list) list)
               *)
              | Some capture_bindings -> Some (r, capture_bindings @ new_bindings)
              )
+         | R.CondAnalysis (mvar, CondEntropyV2) -> (
+             match !hook_pro_entropy_analysis with
+             | None -> failwith "Core entropy plugin not installed"
```

Comment:
Would probably be best to log at error level the rule id. Should be possible to get via `env.rule.id`.